README.md

```diff
@@ -1,189 +1,249 @@
+brew install c2patool
+```
 
-```c2patool image.jpg  -d```
+### Building from source
 
-## Previewing a Manifest
+If you have Rust installed, you can build c2patool from source:
 
-If a path to a json config file is given,
-the tool will generate a new manifest using the values given in definition
-this will display the results but not save anything unless an output (-o) is specified
+```shell
+git clone git@github.com:contentauth/c2patool.git
+cargo build
+```
 
-```c2patool sample/config.json```
+## Usage
 
-The config json can also be passed on the command line as string using the -c --config option
+### Displaying manifest data
+
+Invoking the tool with a path to an image file will output a JSON report of the manifests contained in the file. Supported file formats are `jpeg` and `png`. 
 
 ```shell
-c2patool -c '{"assertions": [{"label": "org.contentauth.test", "data": {"name": "Jane Doe"}}]}'
+c2patool image.jpg
 ```
- 
-## Creating a new output image
 
-A file path for creating an output file with any added claim data  
-If the output file already exists, any C2PA data in that file will be replaced and the image maintained  
-If the output file doesn't exist, a parent file must be available for a source image  
-If you are not changing an image and just adding C2PA data, use an existing output file and no parent  
-If you have edited an image and want to add C2PA data to it, pass the original as the parent
-and put the edited file at the output location to have the C2PA data added.
+#### Detailed manifest report
 
-```c2patool sample/config.json -o output.jpg```
-## Overriding the parent file
+The `-d` option will output a detailed JSON report of the internal C2PA structure.
 
-When using a json file, the parent file can be specified by passing -p or --parent with the path to the file
-This allows adding the same manifest data to different source images
+```shell
+c2patool image.jpg  -d
+```
 
-## Working with .c2pa manifest files
+### Previewing a manifest
 
-If the extension of the output file is '.c2pa' a standalone manifest store will be written 
+If a path to a JSON config file is given, the tool will generate a new manifest using the values given in the definition. By default, this will print the results to stdout. You can save the results to a file by using the `-o` flag, followed by the path of the destination file.
 
-```c2patool claim_image.jpg -o manifest.c2pa```
+```shell
+# output to screen
+c2patool sample/config.json
 
-These .c2pa manifest files can be read by claim tool and will generate reports.
+# save to file
+c2patool sample/config.json -o results.txt
+```
 
-```c2patool manifest.c2pa```
-## Setup
+The config json can also be passed on the command line as string using the -c --config option
+
+```shell
+c2patool -c '{"assertions": [{"label": "org.contentauth.test", "data": {"name": "Jane Doe"}}]}'
+```
+ 
+### Adding a manifest to a file
+
+#### Setup
+
+##### Creating and using an X.509 certificate
```

Comment:
We may want to move all the cert creation documentation to a separate readme, or at least to the bottom of this readme. We will not longer require a user to go through this process just to try out the tool and make test manifests since we are now providing pre-built certs that can be used by default. So you don't need to have openssl and the rest unless you want to create your own official certs, and that is much more involved process.